.prettierrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+	trailingComma: 'es5',
```

Comment:
note: in v3 the default changes to "all"

I've locked this to the v2 default to reduce the diff in this pull request since as an option it will be very easy to do a follow-up PR switching this default if desired